src/routes/events/+page.server.ts

```diff
@@ -1,17 +1,21 @@
 import { getFromCMS } from '$lib/utils.js';
 
 // needs to concat and format this text
+//Todo: figure out why its not grabbing the links and images
 const eventQuery = `*[_type == "events"]{
   name,
   category,
   date,
   location,
   description,
+  link, 
+  payment, 
+  image,
   "lastUpdated": _updatedAt,
 }`;
```

Comment:
Try:

```suggestion
const eventQuery = `*[_type == "events"]{
  name,
  category,
  date,
  paylink,
  reglink,
  "thumbnail": thumbnail.asset->url+"?h=1200&fm=webp",
  description,
  "lastUpdated": _updatedAt,
}
```

You can change the keys if you want!